src/containerapp/azext_containerapp/_params.py

```diff
@@ -117,6 +117,12 @@ def load_arguments(self, _):
         c.argument('environment_name', options_list=['--environment'], help="The environment name.")
         c.argument('resource_group_name', arg_type=resource_group_name_type, id_part=None)
 
+    with self.argument_context('containerapp env', arg_group='Custom Domain') as c:
+        c.argument('certificate_identity', options_list=['--custom-domain-certificate-identity', '--certificate-identity'], 
```

Comment:
>--custom-domain-certificate-identity [](http://example.com/codeflow?start=58&length=36)

too long and it is already meaningful with --certificate-identity.  recommend to use --certificate-identity and -i

Reply:
just for align, please confirm whether we should change it